docs/dg/dev/guidelines/performance-guidelines/general-performance-guidelines.md

```diff
@@ -329,6 +329,54 @@ When using Gateway for Twig rendering–for example, for sending emails–you ca
 1. Update `spryker/twig` to version `3.28.0` or higher.
 2. In `src/Pyz/Zed/Application/ApplicationDependencyProvider.php:getBackendGatewayApplicationPlugins()` replace `TwigApplicationPlugin()` with `TwigGatewayApplicationPlugin()`.
 
+## Order Placement Performance
+You can place order items in batches to improve performance when the cart contains many items.
+
+#### Prerequisites
+1. Update the spryker/sales package and its dependencies to version [spryker/sales:^11.63.0](https://github.com/spryker/sales/releases/tag/11.63.0) .
+2. Add the following configuration to `src/Pyz/Zed/Sales/SalesConfig.php`:
+```php
+<?php
+
+namespace Pyz\Zed\Sales;
+
+use Spryker\Zed\Sales\SalesConfig as SprykerSalesConfig;
+
+class SalesConfig extends SprykerSalesConfig
+{
+    /**
+     * Returns the column that stores a unique identifier for each order item.
+     */
+    public function getItemHashColumn(): string
+    {
+        return 'OrderItemReference';
+    }
+}
+
+```
+#### Choose a Unique Column
+The example above uses the `OrderItemReference` column, which Spryker provides out of the box. You may specify a different column, but the chosen column must:
+- Contain a unique value for every order item in the database.
+- Be generated before the order is saved.
+
+#### Enable `OrderItemReference` Generation (if required)
```

Comment:
How do I know if it's required?

Reply:
i mean, if it's not enabled already, and if a customer decided to use this instead of any custom field
should I change it?

Reply:
If I decide to use another field, will I need to enable generation for that field instead of OrderItemReference?

Reply:
yes, you can use any custom field
I see what is a confusion there, in my original PR there was the next line to explain if required or not `If your project does not yet generate the OrderItemReference value, complete these steps:`
I would suggest to bring it back, idea is that we have it OOTB, but we cannot be sure that each project have it